tests/unit-tests/core/main-class.php

```diff
@@ -75,4 +75,15 @@ public function test_wc_class_instances() {
 		$this->assertInstanceOf( 'WC_Payment_Gateways', $this->wc->payment_gateways() );
 		$this->assertInstanceOf( 'WC_Checkout', $this->wc->checkout() );
 	}
+	
+	public function test_user_agent_header( $webhook = '' ) {
+		if ( ! empty( $webhook ) ) {
```

Comment:
This `if` block will never run. This is better written as two separate test functions, `test_user_agent_header` and `test_user_agent_header_webhook` for example.